vault/resource_ldap_secret_backend_static_role_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccLDAPSecretBackendStaticRole(t *testing.T) {
+	var (
+		path                  = acctest.RandomWithPrefix("tf-test-ldap-static-role")
+		bindDN, bindPass, url = testutil.GetTestLDAPCreds(t)
+		resourceType          = "vault_ldap_secret_backend_static_role"
+		resourceName          = resourceType + ".role"
+		username              = "alice"
+		dn                    = "cn=alice,ou=users,dc=example,dc=org"
+		rotationPeriod        = "60"
+		updatedUsername       = "bob"
+		updatedDN             = "cn=bob,ou=users,dc=example,dc=org"
+		updatedRotationPeriod = "120"
+	)
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion112)
+		},
+		CheckDestroy: testCheckMountDestroyed(resourceType, consts.MountTypeLDAP, consts.FieldPath),
+		Steps: []resource.TestStep{
+			{
+				Config: testLDAPSecretBackendStaticRoleConfig(path, bindDN, bindPass, url, username, dn, username, rotationPeriod),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDN, dn),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldUsername, username),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldRotationPeriod, rotationPeriod),
+				),
+			},
+			{
+				Config: testLDAPSecretBackendStaticRoleConfig(path, bindDN, bindPass, url, updatedUsername, updatedDN, updatedUsername, updatedRotationPeriod),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDN, updatedDN),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldUsername, updatedUsername),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldRotationPeriod, updatedRotationPeriod),
+				),
+			},
+			testutil.GetImportTestStep(resourceName, false, nil, consts.FieldPath, consts.FieldRole, consts.FieldDisableRemount),
```

Comment:
Since we removed `MustAddMountMigrationSchema` from the role resource, the `disable_remount` field is no longer part of our schema, so it can be removed from the ignore list
```suggestion
			testutil.GetImportTestStep(resourceName, false, nil, consts.FieldPath, consts.FieldRole),
```